src/Stratis.Bitcoin.Features.Miner/Staking/PosMinting.cs

```diff
@@ -1134,20 +1135,23 @@ public Models.GetStakingInfoModel GetGetStakingInfoModel()
         /// <summary>
         /// Checks whether the coinstake should be split or not.
         /// </summary>
-        /// <param name="stakedUtxosCount">Number of non-empty UTXOs currently at stake.</param>
-        /// <param name="amountStaked">Total amount currently at stake</param>
-        /// <param name="coinValue">Value of the coin we are considering to split</param>
-        /// <param name="splitFactor">number of time we want to split the coin</param>
-        /// <param name="chainHeight">Current height of the chain</param>
+        /// <param name="stakedUtxosCount">Number of UTXOs that the wallet could stake, if coin base maturity and stake minimum confirmations were not taken into account.</param>
+        /// <param name="amountStaked">Total amount currently at stake.</param>
+        /// <param name="coinValue">Value of the coin we are considering to split.</param>
+        /// <param name="chainHeight">Current height of the chain.</param>
         /// <returns><c>true</c> if the coinstake should be split, <c>false</c> otherwise.</returns>
         /// <remarks>
-        /// The rule is that we want to be splitting coins until they are individually around 100 Strats,
-        /// but aiming not to exceed a total amount of coins bigger than (<c><seealso cref="requiredCoinAgeForStaking"/></c>>+1)*<c><seealso cref="CoinstakeSplitLimitMultiplier"/></c>.
+        /// We do not split a coin if the value of new coins after the split would be less than<see cref="SplitCoinMinSize" />.Because we split the coin to multiple outputs defined by split factor, we only consider coins with value at least <see cref="SplitCoinMinSize" /> * < see cref="SplitFactor" />.
```

Comment:
space after `than` before `<see` and after `/>.` before `Because`

and no space between `< see`